### ISSUE TITLE
Help with guide

### ISSUE THREAD

**Issue by phr34k0** [wwalker0307/ElasticMARC#2]
Please do a guide for linux users as some of us would want this installed on our servers; eg, Digital Ocean. Thanks!

**Comment by wwalker0307** [comment]
> No. Linux has no Powershell whatsoever. Goodness.. Nvm. It's cool. Lel

Yes, I am well aware that PowerShell is a Windows only language.  I'm sorry I wasn't crystal clear in stating that the script will not work, although I never specifically mentioned the script, only the config files.

I would be more than happy to collaborate with anyone that has the Linux skills to develop such a script.  Although, while I am at it, my script also only works with Exchange mailboxes so....apologies to all the other mail servers running on non-Exchange applications.
